packages/hash/integration/src/tests/model/knowledge/entity.model.test.ts

```diff
@@ -37,53 +39,54 @@ describe("Entity CRU", () => {
   let textDataTypeModel: DataTypeModel;
   let namePropertyTypeModel: PropertyTypeModel;
   let favoriteBookPropertyTypeModel: PropertyTypeModel;
-  let linkTypeFriend: LinkTypeModel;
+  let linkEntityTypeFriend: EntityTypeModel;
```

Comment:
can we rename this to `friendLinkEntityTypeModel and it should be `LinkEntityTypeModel` right?

Reply:
Addressed in https://github.com/hashintel/hash/pull/1435/commits/f8c5ec2316c18b2d772803492aae0ebb11391dd8

Reply:
Sorry the naming is still inconsistent. You've written `${name}{type}Model` for the others, but for this you've got `${type}${name}Model` which is quite strange.